.github/workflows/packaging.yml

```diff
@@ -132,8 +132,9 @@ jobs:
           popd
 
       - name: Upload Go binaries
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
+          name: go-${{ matrix.os }}
```

Comment:
v4 errors on default when clobbering files of the same name, as this is a matrix job it was happening.